src/main/java/io/pinecone/clients/Inference.java

```diff
@@ -1,20 +1,22 @@
 package io.pinecone.clients;
 
-import org.openapitools.control.client.ApiClient;
-import org.openapitools.control.client.ApiException;
-import org.openapitools.control.client.api.InferenceApi;
-import org.openapitools.control.client.model.EmbedRequest;
-import org.openapitools.control.client.model.EmbedRequestInputsInner;
-import org.openapitools.control.client.model.EmbedRequestParameters;
-import org.openapitools.control.client.model.EmbeddingsList;
-
-import java.util.List;
-import java.util.Map;
+import io.pinecone.configs.PineconeConfig;
```

Comment:
Since Inference is broken out of control plane, the ApiClient for control plane can't be used as done before